http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/PubsubConfig.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+final case class PubsubConfig(
```

Comment:
think I had a previous comment about removing this datatype and just use the `PublisherConfig` directly

Reply:
yeah, I was going to, but i need the queueSize, which isn't in the PublisherConfig